projects/aca-content/src/lib/components/search/search-results/search-results.component.spec.ts

```diff
@@ -54,6 +54,7 @@ describe('SearchComponent', () => {
   const searchRequest = {} as SearchRequest;
   let params: BehaviorSubject<any>;
   let showErrorSpy: jasmine.Spy;
+  let showInfoSpy: jasmine.Spy;
```

Comment:
I know that above it's written like that but Spy is generic type so ideally you should set types for mocked function inside ```<>```. You can search for examples in other spec files how to set this